### PR TITLE
Add Isaac to HoF

### PR DESCRIPTION
**Pull request by breilly2**

### CODE REVIEW

**Review by zarsky-broad — APPROVED** [review]

File: src/pages/HoF.js [hunk 1]
```diff
@@ -48,6 +48,10 @@ const HallOfFame = () => {
         name: 'Pete Santos',
         dates: 'May 2019 - March 2020',
         pic: 'https://github.com/petesantos.png?size=200'
+      }, {
+        name: 'Isaac Zarsky',
+        dates: 'February 2018 - May 2020',
```
Comment (breilly2) [inline src/pages/HoF.js:53]: Is February right or did this all technically start before then?

Comment (zarsky-broad) [reply src/pages/HoF.js:53]: Lemme see...

**Review by s-rubenstein — APPROVED** [review]
❤️ 😢 :heart:

File: src/pages/HoF.js (hunk 1 above)
Comment (zarsky-broad) [reply src/pages/HoF.js:53]: Earliest meeting I can find is "UI Tech Stack Discussion", 3pm on 2/15/18; I think that was just before the team officially started to exist; the proof of concept repo I made using JS was later that month.

**Pull request merged**